cibuildwheel/linux.py

```diff
@@ -60,6 +63,8 @@ def build(project_dir, output_dir, test_command, test_requires, test_extras, bef
             {environment_exports}
 
             for PYBIN in {pybin_paths}; do
+                "$PYBIN/pip" install --upgrade pip setuptools wheel delocate
```

Comment:
`auditwheel` requires `wheel` 0.31 not 0.34. 

why you install delocate here?

Reply:
> `auditwheel` requires `wheel` 0.31 not 0.34.

Yes, but we need `wheel` 0.34 to get the correct PyPy tags/wheel names.

> why you install `delocate` here?

Because I'm silly and copied too much. Thanks for noticing!